acct-group/sgx/sgx-0.ebuild

```diff
@@ -0,0 +1,8 @@
+# Copyright 2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit acct-group
+
+ACCT_GROUP_ID=405
```

Comment:
This `sgx` group seems to be missing from https://github.com/flatcar-linux/baselayout/blob/flatcar-master/baselayout/group. I don't know if this is important, though.